packages/did-core-test-server/suites/implementations/did-example-didwg.json

```diff
@@ -83,6 +85,10 @@
         "capabilityDelegation": ["#key-0"],
         "keyAgreement": ["#key-1"]
       },
+      "dmRepresentationSpecificEntries": {
+        "@context": ["https://www.w3.org/ns/did/v1"]
+      },
+      "representation": "{\"@context\":[\"https://www.w3.org/ns/did/v1\"],\"id\":\"did:example:123\",\"verificationMethod\":[{\"id\":\"did:example:123#key-0\",\"type\":\"Ed25519VerificationKey2018\",\"controller\":\"did:example:123\",\"publicKeyBase58\":\"6fioC1zcDPyPEL19pXRS2E4iJ46zH7xP6uSgAaPdwDrx\"},{\"id\":\"did:example:123#key-1\",\"type\":\"X25519KeyAgreementKey2019\",\"controller\":\"did:example:123\",\"publicKeyBase58\":\"FxfdY3DCQxVZddKGAtSjZdFW9bCCW7oRwZn1NFJ2Tbg2\"}],\"authentication\":[\"#key-0\"],\"assertionMethod\":[\"#key-0\"],\"capabilityInvocation\":[\"#key-0\"],\"capabilityDelegation\":[\"#key-0\"],\"keyAgreement\":[\"#key-1\"]}",
```

Comment:
Could `didDocumentStream` be used instead of `representation`, for consistency with DID Resolution (`resolveStream` function)?

Reply:
I think `representation` is more accurate, since `didDocumentStream` is only mentioned in the DID Resolution section, but this data here seems to be about production and consumption of representations.

Reply:
Keep as is for now, raise an issue, and change it later if there is stronger consensus to change it to something else.

TODO: Create issue, link issue, merge PR.

Reply:
Tracking issue created here: https://github.com/w3c/did-test-suite/issues/63